models/intermediate/int_jira__issue_comments.sql

```diff
@@ -1,4 +1,4 @@
-{{ config(enabled=var('jira_include_comments', True)) }}
+{{ config(enabled=var('jira_include_comments', False if target.type == 'redshift' else True)) }}
```

Comment:
Taking a closer look at this now, since you were able to identify that the string_agg was likely the only issue I think we may be going overboard with turning off the entirety of this model by default for Redshift users. Especially when there is still some valuable information here (with the `count_comments` field).

Additionally, if we disable these models by default then we don't end up using the `stg_jira__comment` staging model at all in the downstream transformations. 

![image](https://github.com/user-attachments/assets/1c65e11f-caf4-485e-b1d2-56dfe5524a75)

Can we instead adjust this model in particular to not disable it entirely, but just provide the conditional on the string_agg to include or not include based on the `jira_using_comments` variable. This way we only ignore this problematic field for Redshift while still keeping the other aggregate comment field which could be useful in downstream models.

Reply:
We would then need to make slight adjustments in the `int_jira__issue_join` model to account for this dynamic functionality.

Reply:
@fivetran-joemarkiewicz Thanks for the explanation! I updated accordingly, and it's ready for re-review!